tools/pontoon/README.md

```diff
@@ -5,4 +5,7 @@ This document covers how to work with [Pontoon](https://pontoon.mozilla.org/) fr
 * [User accounts and settings](users.md).
 * [Teams and projects](teams_projects.md).
 * [Search and filters](search_filters.md).
-* Translating strings and best practices.
+* [The Translation User Interface](ui.md)
+* [Translating in Pontoon](translate.md)
+* Translation Resources in Pontoon
+* Example Work flows
```

Comment:
You're right. I succumbed to the almighty spellchecker when I should have stood my ground...